numpyro/distributions/discrete.py

```diff
@@ -74,7 +86,7 @@ def _rvs(self, n, p):
         return device_put(sample)
 
     def _logpmf(self, x, n, p):
-        n, p = _promote_dtypes(n, p)
+        x, n, p = _promote_dtypes(x, n, p)
```

Comment:
Not relevant to this PR, but I just noticed that we aren't normalizing logits and assuming that the logits passed in are normalized i.e. `sum(np.exp(logits)) == 1`. This won't be true when we pass in logits from NNs.

Reply:
I think that we just need to normalize logits in multivariate distributions and I already did it.